src/halmos/sevm.py

```diff
@@ -1323,6 +1330,27 @@ def arith2(self, ex: Exec, op: int, w1: Word, w2: Word, w3: Word) -> Word:
         else:
             raise ValueError(op)
 
+    def resolve_address_alias(self, ex: Exec, target: Address) -> Address:
+        if target in ex.code:
+            return target
+
+        if target not in ex.alias:
+            for addr in ex.code:
+                if (
+                    # must equal
+                    ex.check(target != addr) == unsat
+                    # ensure existing path condition is feasible
+                    and ex.check(target == addr) != unsat
```

Comment:
we take `unknown` as `may be feasible` here?

Reply:
yes.  not only here, but throughout the path exploration phase, we prune only unsat paths.